tests/test-war-smoke/pom.xml

```diff
@@ -45,6 +45,7 @@
           <artifactId>servlet-api</artifactId>
         </exclusion>
       </exclusions>

```

Comment:
https://github.com/google/google-oauth-java-client/issues/145 has been fixed. So the `servlet-api` exclusion should no longer be necessary.